frontend/src/services/data.service.js

```diff
@@ -1,97 +1,38 @@
 import axios from "axios";
-import Cookies from "universal-cookie";
-import * as jwt from "jsonwebtoken";
-
-const cookies = new Cookies();
-const baseUrl =
-    process.env.NODE_ENV === "development"
-        ? "http://localhost:5000/api"
-        : "https://suggestit.chalmers.it/api";
-const authCookieName = "AUTH_TOKEN";
 
 export const updateSuggestions = setter =>
     axios
-        .get(`${baseUrl}/`, {
-            headers: {
-                Authorization: cookies.get(authCookieName),
-            },
-        })
```

Comment:
This is a **big red flag!**. 
Your cookies must be HTTP only! This is a huge security issue. You should not have to touch your cookies at all. 

Cookies in production must always be `secure`, `httponly` and `same-site: strict`.

Reply:
When attaching cookie in the backend, cookies are set to secure and httponly but does not use the correct domain.

See comment below